app/controllers/auth_controller.rb

```diff
@@ -0,0 +1,48 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+class AuthController < ApplicationController
+    def index
+    end
+
+    def signup
+        first_name = params[:first_name]
+        last_name = params[:last_name]
+        email = params[:email]
+        password = params[:password]
+
+        new_user = User.new(first_name: first_name, last_name: last_name, email: email)
+        new_user.save
+
+        uri = URI("https://identitytoolkit.googleapis.com/v1/accounts:signUp?key=#{ENV['FIREBASE_API_KEY']}")
```

Comment:
This URI string gets repeated quite a bit. Can you abstract that as a global variable for all methods in the class to access?